store/sdk/src/test/java/org/apache/carbondata/sdk/file/CSVCarbonWriterTest.java

```diff
@@ -543,4 +543,42 @@ public void testWritingAndReadingArrayOfFloatAndByte() throws IOException {
     }
   }
 
+  @Test
+  public void testWithTableProperties() throws IOException {
```

Comment:
what's the use of this test case ? how would you validate table property is working or not ?